web/packages/teleport/src/services/apps/types.ts

```diff
@@ -20,6 +20,8 @@ import { AwsRole } from 'shared/services/apps';
 
 import { ResourceLabel } from 'teleport/services/agents';
 
+import type { SamlServiceProviderPreset } from 'teleport/Discover/SelectResource/types';
+
```

Comment:
IMO this type should live in this file. We rather want `services` to be our source of truth when it comes to "core" types.
Now we have a some kind of a circular import: `teleport/services/apps` imports `teleport/Discover/SelectResource/types` and in turn  `teleport/Discover/*` imports `teleport/services/apps`.

Reply:
Yaa true. The concrete types for SAML was added in services on `e` repo - https://github.com/gravitational/teleport.e/blob/master/web/teleport/src/services/idp/types.ts

This one started its place in Discover but I should have moved that to `oss/services` from the beginning. What I am thing now is to move the `e` types to `oss` and we will have a single type file for saml idp and service provider resources. That I will revisit some time later if that is ok with you?

Reply:
>What I am thing now is to move the e types to oss and we will have a single type file for saml idp and service provider resources.

👍 

>That I will revisit some time later if that is ok with you?

👍

Reply:
Thanks.